docs/guide/persistence_cache.md

```diff
@@ -108,7 +97,23 @@ services:
 
 ##### Redis Cluster
 
-It is possible to set up and use Redis as a cluster. This configuration is more efficient and reliable for large installations. Redis Cluster can be configured in two ways, the first using [create-cluster script](https://redis.io/topics/cluster-tutorial) and the second using [Redis Sentinel](https://redis.io/topics/sentinel). If you use Platform.sh Enterprise you can benefit from the Redis Sentinel across three nodes for greater fault tolerance. Platform.sh Professional and lower versions offer Redis in single instance mode only. Configuration on eZ Platform / Symfony stays the same regardless of the Redis version, single instance mode or cluster mode.
+Configuration of several Redis instances can be done in several ways, as Persistence cache depends on all involved web servers seeing the same view of the cache as it's shared, the following applies:
```

Comment:
Nitpick:
`Configuration` -> `The configuration`

As for me this sentence is a little bit too complicated, WDYT about putting it like this:
```
Configuration of several Redis instances can be done in several ways. Persistence cache depends on all involved web servers, each of them seeing the same view of the cache because it's shared among them. The following configurations are possible:
```